packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/Controls.tsx

```diff
@@ -4,6 +4,7 @@ import { isString } from '@aws-amplify/ui';
 import { LocationData, useAction } from '../../context/actions';
 import { useControl } from '../../context/control';
 import { parseLocationAccess } from '../../context/navigate/utils';
+import { Title } from '../../composables/Title';
```

Comment:
Should we use `TitleControl` with `ControlsContextProvider`  in all the views instead of the composable component.  
[Reference PR](https://github.com/aws-amplify/amplify-ui/pull/5939/files#diff-2a298ab664ed59a235a070ed3e4ca2e5045c4383dadd8afb5ceda79cc2c142a6R15)